app/_how-tos/store-the-gateway-database-credentials-with-aws-secrets-manager.md

```diff
@@ -0,0 +1,144 @@
+          - `secretsmanager:CreateSecret`
+          - `secretsmanager:PutSecretValue`
+          - `secretsmanager:GetSecretValue`
+        - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) installed
+        
+        You will also need the following authentication information to connect your AWS Secrets Manager with {{site.base_gateway}}:
+        - Your access key ID
+        - Your secret access key
+        - Your session token
+        - Your AWS region, `us-east-1` in this example
+
+        Create environment variables to store these credentials:
+        ```sh
+        export AWS_ACCESS_KEY_ID=your-aws-access-key-id
+        export AWS_SECRET_ACCESS_KEY=your-aws-secret-access-key
+        export AWS_SESSION_TOKEN=your-aws-session-token
+        export AWS_REGION="us-east-1"
+        ```
+      icon_url: /assets/icons/aws.svg
+
+cleanup:
+  inline:
+    - title: Clean up AWS resources
+      include_content: cleanup/third-party/aws
+      icon_url: /assets/icons/aws.svg
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg 
+---
+
+## 1. Create a Docker network
+```sh
+docker network create kong-net
+```
+The Docker network will be used for communication between {{site.base_gateway}} and the database.
+
+## 2. Run the database
+Create the `kong-database` container for the PostgreSQL database: 
+```sh
+docker run -d --name kong-database \
+ --network=kong-net \
+ -p 5432:5432 \
+ -e "POSTGRES_USER=admin" \
+ -e "POSTGRES_PASSWORD=password" \
+ postgres:9.6
+```
+The username and password specified in this command are the PostgreSQL master credentials.
+
+## 3. Create environment variables
+Define the username and password to use to connect {{site.base_gateway}} to the database and store them in environment variables.
+```sh
+export KONG_PG_USER=kong
+export KONG_PG_PASSWORD=KongFTW
+```
+
+## 4. Create a database user
+Create a user in the PostgreSQL container, using the credentials defined in the previous step:
+```sh
+docker exec -it kong-database psql -U admin -c \
+ "CREATE USER ${KONG_PG_USER} WITH PASSWORD '${KONG_PG_PASSWORD}'"
+```
+
+## 5. Create a database
+Create a database named `kong`, with the user you created as the owner:
+```sh
+docker exec -it kong-database psql -U admin -c "CREATE DATABASE kong OWNER ${KONG_PG_USER};"
+```
+
+## 6. Create a secret in AWS Secrets Manager
+Use AWS CLI to create a new secret named `kong-gateway-database` containing the username and password you defined:
+```sh
+aws secretsmanager create-secret --name kong-gateway-database \
+ --description "Kong GW Database credentials" \
+ --secret-string '{"pg_user":"'${KONG_PG_USER}'","pg_password":"'${KONG_PG_PASSWORD}'"}'
+```
+
+## 7. Run the Kong migrations
+```sh
```

Comment:
Add a sentence here about what `kong migrations` is doing in this instance. 

I tried to find information on it, and it was sparse. Looks like it can be used to migrate from OSS to EE.